### PR TITLE
Swap on-my-own const for using the existing Personal const

### PR DESCRIPTION
**Pull request by marcellamaki**
prevent state management snafus

## Summary

…

## References

…

## Reviewer guidance

…

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by radinamatic** [comment]
After installing the APK asset from this PR on my Android 9 device and attempted the _on-my-own_ setup, I encountered the [same error as in 4. reported previously](https://github.com/learningequality/kolibri/issues/11487).

However, once tapped the link _Start over_ in the error, the setup was eventually completed 🤷🏽‍♀️ 
I repeated the workflow with another, even older device (Android 8), and had no issues with the _on-my-own_ setup, so not sure what's the issue. Attaching logs: [kolibri.txt](https://github.com/learningequality/kolibri/files/13311980/kolibri.txt)

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/HowAreYouUsingKolibri.vue [hunk 1]
```diff
@@ -36,18 +36,21 @@
...
       isOnMyOwnSetup() {
-        return this.selected === UsePresets.ON_MY_OWN;
+        return this.selected === Presets.PERSONAL;
       },
       UsePresets() {
         return UsePresets;
       },
+      Presets() {
+        return Presets;
+      },
```
Comment (rtibbles) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/HowAreYouUsingKolibri.vue:50]: I noticed an error in the console while going through the on my own flow:

```
vuex.esm.js:460 [vuex] unknown mutation type: SET_FACILITY_PRESET
commit @ vuex.esm.js:460
boundCommit @ vuex.esm.js:409
handleContinue @ HowAreYouUsingKolibri.vue:60
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ OnboardingStepBase.vue:200
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:181
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
```

Looks like the call to the `SET_FACILITY_PRESET` mutation below is calling a non-existent mutation?

**Review by rtibbles — APPROVED** [review]
This does rather feel like a double hack - both removing any use of the ON_MY_OWN value, replacing it with PERSONAL, and also adding the computed prop at the end to make sure we always set the PERSONAL preset in the on my own case.

However, this close to release, better safe than sorry.

One possible cleanup, as I saw an error, that seems to be pre-existing, when going through the flow.

**Comment by rtibbles** [comment]
Docs build issue is fixed on main branch.

**Pull request merged**